guide/de/ch_reports.xml

```diff
@@ -0,0 +1,1207 @@
+ zugewiesen sind, in absteigender Reihenfolge nach dem Wert zum Ende der
+ aktuellen Abrechnungsperiode angeordnet.</para>
+<para>Verschiedene Einstellungen in diesem Bericht können die enthaltenen
+ Informationen stark beeinflussen.</para>
+<itemizedlist>
+<listitem><para>Auf der Registerkarte "Konten" ändert die Option "Konten
+ anzeigen bis Ebene", wie der Bericht die Kontosummen aggregiert. Ändern
+ Sie diesen Wert, um Informationen auf tieferen Ebenen der Kontostruktur
+ zu sehen.</para></listitem>
+<listitem><para>Auf der Registerkarte "Anzeige" werden mit der Option "Maximale Abschnitte" mehr Abschnitte im Diagramm angezeigt, sodass Informationen für mehr Konten angezeigt werden können.</para></listitem>
+<listitem><para>Auf der Registerkarte "Allgemein" kann die Option "Preisquelle" den berichteten Wert der verschiedenen im Bericht enthaltenen Wertpapiere erheblich beeinflussen.</para></listitem>
+</itemizedlist>
+</sect3>
+
+<sect3 id="rpt_incstatement">
+<title>Ergebnisrechnung </title>
+<para>Dieser Bericht listet die Summen der Aufwands- und Ertragskonten
+für einen bestimmten Zeitraum auf. Standardmäßig zeigt er alle Aufwands-
+und Ertragskonten bis hinunter zu 3 Ebenen von Unterkonten für die
+aktuelle Finanzperiode.</para>
+<para>Eine Gewinn- und Verlustrechnung wird auch als "Gewinn- und Verlust"-Bericht oder "Umsatzrechnung" bezeichnet.</para>
+<para>In früheren Versionen von &app; hieß dieser Bericht "Gewinn &amp;
+ Verlust", aber mit Version 2 wurde der Bericht in "Gewinn- und
+ Verlustrechnung" umbenannt, um eine gängigere Buchhaltungsterminologie
+ zu verwenden.</para>
+<para>Die Gewinn- und Verlustrechnung zeigt, woher das Geld kommt und
+wohin es in einem bestimmten Zeitraum fließt.</para>
+</sect3>
+
+<sect3 id="rpt_incdayoweek">
+<title>Ertrag pro Wochentag</title>
+<para>Erträge pro Wochentag zeigt ein Tortendiagramm mit den Summen für
+ausgewählte Ertragskonten, summiert nach dem Wochentag der Buchung. Mit
+den Berichtsoptionen können Sie einige Anpassungen vornehmen (z. B.
+Konten, Anzeigeoptionen und den Datumsbereich), aber die Kontenauswahl
+erlaubt nur die Auswahl von Ertragskonten. Der Bericht aggregiert Buchungen
+nach Wochentag, nicht nach anderen Zeiträumen oder Kategorien. Aufgrund
+dieser Einschränkungen kann der Bericht als Demonstration oder als
+Beispiel für jemanden betrachtet werden, der den Quellcode für die
+Erstellung eines nützlichen benutzerdefinierten Berichts untersuchen
+möchte..</para>
+</sect3>
+
+<sect3 id="rpt_trialbal">
+<title>Rohbilanz</title>
+<para>Die Rohbilanz listet die Endsalden aller Konten zu einem bestimmten
+ Datum auf. Sie wird typischerweise am Ende einer Buchhaltungsperiode
+ ausgeführt und dient in erster Linie dazu, sicherzustellen, dass die
+ Summe aller Belastungen gleich der Summe aller Gutschriften ist.</para>
+</sect3>
+<sect3 id="rpt_gst_statement">
+  <title>Einkommens- und Umsatzsteuererklärung (GST)</title>
```

Comment:
Australische …
Chris glaubt, sie klappt auch in GB. Ob sie außerhalb des angelsächsischen Sprach- und Kulturraums Sinn macht, müßte mal jemand prüfen.
Ich denke man sollte sie mit TXF, ElStEr, … gruppieren
und auch in der Programm-Übersetzung des Titel/Namen deuttlich machen.